hack/internal/pkg/version/version.go

```diff
@@ -0,0 +1,150 @@
+package version
```

Comment:
issue: Let's drop this `version` package because some of the values that are set here no longer make sense since we're not using `goreleaser` to build these binaries.